content/projects/gsoc/2019/project-ideas/automatic-spec-generator-for-jenkins-rest-api.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: gsocprojectidea
+title: "Automatic Specification Generator for Jenkins REST API"
+goal: "Find and implement the extraction of the REST APIs from the sources and generate and publish the REST APIs respective documentation"
+category: Plugins
+year: 2019
+status: draft
+skills:
+- Java
+- CSS
```

Comment:
I do not see how CSS and HTML tags address the project idea. REST API and Swagger could be better

Reply:
@oleg-nenashev I transcribed what @martinda wrote in the proposal; could you comment in Google Doc? Thanks.